style.css

```diff
@@ -205,49 +210,63 @@ img {vertical-align: middle;}
   to {opacity: 1}
 }
 
-/* On smaller screens, decrease text size */
+/* On smaller screens,redcrease text size */
 @media only screen and (max-width: 300px) {
   .prev, .next,.text {font-size: 11px}
 }
 
 /* Abouts_us (index) */
 
 .about_us_A  {
-	background-color: #CABDBD;
-	grid-column-start: 1;
-	grid-column-end: 3;
+	margin-top: 50px;

```

Comment:
I decided to add margins to make the "About_Us" section look more like box and different from the rest of the page, for an aesthetic effect - Refinement